Mathlib/Analysis/Normed/Group/SemiNormedGroup.lean

```diff
@@ -117,24 +162,31 @@ def SemiNormedGroup₁ : Type (u + 1) :=
 
 namespace SemiNormedGroup₁
 
-instance : CoeSort SemiNormedGroup₁ (Type u) :=
-  Bundled.hasCoeToSort
+--Porting Note: hasCoeToSort doesn't exist anymore
+--instance : CoeSort SemiNormedGroup₁ (Type u) :=
+--  Bundled.hasCoeToSort
+
+instance : CoeSort SemiNormedGroup₁ (Type _) where
+  coe X := X.α
+
 
-instance : LargeCategory.{u} SemiNormedGroup₁ where
-  hom X Y := { f : NormedAddGroupHom X Y // f.NormNoninc }
-  id X := ⟨NormedAddGroupHom.id X, NormedAddGroupHom.NormNoninc.id⟩
-  comp X Y Z f g := ⟨(g : NormedAddGroupHom Y Z).comp (f : NormedAddGroupHom X Y), g.2.comp f.2⟩
+--Porting Note: Was explicitly shown before. Is there a problem doing it like this?
+deriving instance LargeCategory for SemiNormedGroup₁
```

Comment:
This has broken the file,`deriving` is finding an instance different to the old one.

Reply:
Thanks! Reverted to there.